esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3434,7 +3453,17 @@ bool CFileSprayEx::onGetSprayTargets(IEspContext &context, IEspGetSprayTargetsRe
     try
     {
         context.ensureFeatureAccess(FILE_SPRAY_URL, SecAccess_Read, ECLWATCH_FILE_SPRAY_ACCESS_DENIED, "Permission denied.");
-
+#ifdef _CONTAINERIZED
+        IArrayOf<IEspGroupNode> sprayTargets;
+        Owned<IPropertyTreeIterator> dataPlanes = queryGlobalConfig().getElements("storage/planes[labels='data']");
+        ForEach(*dataPlanes)
+        {
+            IPropertyTree & plane = dataPlanes->query();
+            const char * name = plane.queryProp("@name");
+            appendGroupNode(sprayTargets, name, name, false /*replicate outputs*/);
```

Comment:
is it correct to pass 'name' to 3rd param (clusterType)  ?
I guess it's a bit meaningless in cloud, it's usually "hthor", "thor", or "roxie" ..
Is it just for display purposes by the client? Perhaps should just be a generic "storage plane"